public/style.css

```diff
@@ -0,0 +1,236 @@
+* {
+  margin:0;
+  padding:0;
+  border:0;
+}
+
+.container {
+  border-width: 1px;
+  border-color: rgb(235, 235, 235);
+  border-style: solid;
+  background: white;
+  font-family: 'Nunito Sans', sans-serif;
+  width: 375px;
+  height: 350px;
+  display: grid;
+  justify-items: stretch;
+  grid-template-columns: 25px 145px 35px 145px 25px;
+  grid-template-rows: 22px 60px 75px 85px 60px 48px;
+}
+
+.calendarContainer {
+  grid-column-start: 2;
+  grid-column-end: 5;
+  grid-row-start: 3;
+  grid-row-end: 4;
+}
+
+.guestsContainer {
+  grid-column-start: 2;
+  grid-column-end: 5;
+  grid-row-start: 4;
+  grid-row-end: 5;
+  align-self: center;
+}
+
+.propertyContainer {
+  grid-column-start: 2;
+  grid-column-end: 5;
+  grid-row-start: 2;
+  grid-row-end: 3;
+
+  border-bottom-color: rgb(235, 235, 235);
+  border-bottom-style: solid;
+  border-bottom-width: 1px;
+  color: rgb(72, 72, 72);
+}
+
+#price {
```

Comment:
Try to use more classes than id's in css, thin about what's happen when you want to have few places with price for example...